nixos/modules/services/databases/mongodb.nix

```diff
@@ -121,6 +120,15 @@ in
       }
     ];
 
+    services.mongodb.package =
+     # Note: when changing the default, make it conditional on
+     # ‘system.stateVersion’ to maintain compatibility with existing
+     # systems!
+     # Remember to change the default version at pkgs/top-level/all-packages.nix too between NixOS versions
+     mkDefault (if versionAtLeast config.system.stateVersion "22.05" then pkgs.mongodb-5_0
```

Comment:
Do you need to put 22.11 since 22.05 is the default ?

Reply:
Indeed, since we missed 22.05 release, tihs should be 22.11.

Since I only with a laptop now, I will delay a bit any change for now but I may give a look at weekend.